src/routes/course.js

```diff
@@ -1,23 +1,34 @@
 const express = require('express');
```

Comment:
I think you'll also need the `body-parser` middleware
Install with `npm iinstall --save body-parser`
and use:
```
const bodyParser = require('body-parser');

app.use(bodyParser.json());
```